corehq/apps/sso/views/enterprise_admin.py

```diff
@@ -125,7 +124,7 @@ def edit_enterprise_idp_form(self):
 
     @property
     def show_remote_user_management(self):
```

Comment:
I think the idea here is to remove the usage of this property as well, as it's no longer relevant

Reply:
Thank you! Fixed in e18c7ac